src/persons/HowManyPersonsLiveTogetherPage.js

```diff
@@ -28,27 +28,25 @@ let HowManyPersonsLiveTogetherPage = props => {
   return (
       <Grid container className='bg-container' justify='center'>
         <Grid item>
+          <Typography variant='headline' gutterBottom>
```

Comment:
Please stick to the design. The button is on the right side. Field is vertically centered.

Reply:
The design in the PSD is totally different, the Field is on the left side and the button on the right side.
I don't know what organization use right now.